wandb/sdk/launch/agent/agent.py

```diff
@@ -167,7 +167,7 @@ def run_job(self, job: Dict[str, Any]) -> None:
         _logger.info("Fetching resource...")
         resource = launch_spec.get("resource") or "local-container"
         backend_config: Dict[str, Any] = {
-            PROJECT_DOCKER_ARGS: {},
+            PROJECT_DOCKER_ARGS: launch_spec.get("docker", {}).get("args", {}),
```

Comment:
Could either value ever be None? If yes, then
```suggestion
            PROJECT_DOCKER_ARGS: (launch_spec.get("docker", {}) or {}).get("args", {}) or {},
```

Reply:
I guess if the user specifically sets it to null in the UI it could be None. Nice catch.